examples/cloudml-support-routing/scripts/predict.py

```diff
@@ -0,0 +1,129 @@
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Makes a prediction using the AutoML Tables model specified in the config.
+Assumes that a model with the specified display name has already been trained,
+and that the features for prediction have been generated. Prediction
+jobs typically take 5-25 minutes to complete, based on the volume of data.
+See https://cloud.google.com/automl-tables/docs/predict-batch for details.
+"""
+
+import logging
+import sys
+
+import google.api_core.exceptions
+from google.cloud import automl_v1beta1 as automl
+from google.cloud import bigquery
+
+import utils
+
+logging.basicConfig(level=logging.DEBUG)
+
+
+def main():
+  """Executes batch prediction using a model trained on AutoML Forecasting.
```

Comment:
This example uses AutoML Tables Forecasting? Or is this a typo and its meant to say AutoML Tables?

Reply:
Good catch. Found a few of these and fixed them.